Development.md

```diff
@@ -91,9 +97,11 @@ PASSWORD=45e0f109-3f56-4b30-9b5a-b0939b8a4c25
 ...
 ```
 
+These secrets can be used in your local rack docker vm for interfacing with AWS.
+
 ## Local Rack Docker VM
 
-A local Rack is started with `convox start` which requires a working Docker environment. To setup a Docker environment, see the [Docker Machine](https://docs.docker.com/machine/) docs. You can then run the project:
+A local rack is started with `convox start` which requires a working Docker machine environment. To setup a Docker environment by creating a Docker machine, see the [Docker Machine](https://docs.docker.com/machine/) docs. You can then run the project:
```

Comment:
here are cases where the "Rack" v. "rack" confused me.

And also fwiw I think a "Docker Machine" environment is unique to a regular "Docker environment" b/c I view the latter as just running one-off containers.